test/jdk/java/awt/FullScreen/FullscreenWindowProps/FullscreenWindowProps.java

```diff
@@ -52,6 +52,10 @@ public void paint(Graphics g) {
                 super.paint(g);
                 g.setColor(Color.GREEN);
                 g.fillRect(0, 0, getWidth(), getHeight());
+                g.setColor(Color.RED);
+                DisplayMode displayMode =
+                        getGraphicsConfiguration().getDevice().getDisplayMode();
+                g.drawString(displayMode.toString(), 100, 100);
```

Comment:
@azvegint 
This is on an unchanged line. The finally block in the test sets `gd.setFullScreenWindow(null);`
Is that sufficient or do we need to additionally restore the original display mode in the finally block?

Reply:
This additional `drawString` was added to make the current display mode more visually distinguishable. It's hard to see just from the window filled with green color.

> The finally block in the test sets gd.setFullScreenWindow(null);
Is that sufficient or do we need to additionally restore the original display mode in the finally block?

Yes, it is sufficient, it is documented here:
https://github.com/openjdk/jdk/blob/78c18cfbcee92ba170810582e238b40b64805e5a/src/java.desktop/share/classes/java/awt/GraphicsDevice.java#L257-L262